pkg/frontend/middleware/log.go

```diff
@@ -176,24 +176,23 @@ func Log(env env.Core, auditLog, baseLog *logrus.Entry) func(http.Handler) http.
 }
 
 func auditTargetResourceData(r *http.Request) (string, string) {
```

Comment:
is the second return value from this function always just r.URL.Path?  If so please simplify so this is just a function which calculates and returns the resourceKind?